ui/admin-portal/src/app/components/util/det-score-validation/det-score-validation.component.ts

```diff
@@ -0,0 +1,40 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {UntypedFormControl} from "@angular/forms";
+@Component({
+  selector: 'app-det-score-validation',
+  templateUrl: './det-score-validation.component.html',
+  styleUrls: ['./det-score-validation.component.scss']
+})
+export class DetScoreValidationComponent implements OnInit{
+
+  @Input() control: UntypedFormControl;
+
+  error: string;
+  regex: RegExp;
+  value: string;
+
+  constructor() { }
+
+  ngOnInit(): void {
+    this.value = this.control.value;
+    this.regex = new RegExp('^(1[0-9]|[2-9][0-9]|1[0-5][0-9]|160)$');
+  }
+
+  update() {
+    if (this.value !== null) {
+      if (!this.regex.test(this.value)) {
```

Comment:
This of course works but personally I'd just test that the value is in the required range i.e. >= 10 and <= 160 for readability